boring/src/pkey.rs

```diff
@@ -93,20 +93,23 @@ impl Id {
 }
 
 /// A trait indicating that a key has parameters.
+#[allow(clippy::missing_safety_doc)]
 pub unsafe trait HasParams {}
 
 unsafe impl HasParams for Params {}
 
 unsafe impl<T> HasParams for T where T: HasPublic {}
 
 /// A trait indicating that a key has public components.
+#[allow(clippy::missing_safety_doc)]
 pub unsafe trait HasPublic {}
 
 unsafe impl HasPublic for Public {}
 
 unsafe impl<T> HasPublic for T where T: HasPrivate {}
 
 /// A trait indicating that a key has private components.
+#[allow(clippy::missing_safety_doc)]
```

Comment:
Please move this into a separate PR, it's not related to iOS support.

Reply:
I'm sorry for mixing changes. Moved this commit into #61.